temporal-normalizer/src/test/java/edu/illinois/cs/cogcomp/temporal/normalizer/tests/TestTemporalChunker.java

```diff
@@ -0,0 +1,365 @@
+                Element rootElement = document.getDocumentElement();
+                NodeList nodeList = rootElement.getChildNodes();
+                for (int i = 0; i < nodeList.getLength(); i++) {
+                    Node currentNode = nodeList.item(i);
+                    if (currentNode.getNodeName().equals("TEXT")) {
+                        testText.add(currentNode.getTextContent());
+                    }
+                    if (currentNode.getNodeName().indexOf("DOCID")!=-1) {
+                        docIDs.add(currentNode.getTextContent());
+                    }
+                    if (currentNode.getNodeName().indexOf("DCT")!=-1) {
+                        Node dctNode = currentNode.getChildNodes().item(0);
+                        NamedNodeMap dctAttrs = dctNode.getAttributes();
+                        for (int j = 0; j < dctAttrs.getLength(); j++) {
+                            if (dctAttrs.item(j).getNodeName().equals("value")) {
+                                DCTs.add(dctAttrs.item(j).getNodeValue());
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+//        te3inputText = new ArrayList<>();
+//        URL te3URL = TestTemporalChunker.class.getClassLoader().getResource(te3ForderName);
+//        File te3Folder = new File(te3URL.getFile());
+//        File[] te3Inputs = te3Folder.listFiles();
+//        for (File file : te3Inputs) {
+//            String testFile = te3URL.getFile() + "/" + file.getName();
+//            byte[] encoded = Files.readAllBytes(Paths.get(testFile));
+//            String fileContent = new String(encoded, StandardCharsets.UTF_8);
+//            te3inputText.add(fileContent);
+//        }
+
+    }
+
+    private static final String textFile = "text/AP_20130322";
+
+    @Test
+    public void testPosPipeline() {
+        String input = null;
+        try {
+            input = LineIO.slurp(textFile);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+            fail(e.getMessage());
+        }
+
+        assertFalse(input.length() == 0);
+
+        AnnotatorService as = null;
+        try {
+            as = PipelineFactory.buildPipeline(ViewNames.POS);
+        } catch (IOException | AnnotatorException e) {
+            e.printStackTrace();
+            fail(e.getMessage());
+        }
+
+        String norm = input.replaceAll("\\n", " ");
+        norm = StringCleanup.normalizeToUtf8(norm);
+
+
+        try {
+            as.createAnnotatedTextAnnotation("test", "test", norm);
+        } catch (AnnotatorException e) {
+            e.printStackTrace();
+            fail(e.getMessage());
+        }
+    }
+
+    @Test public void testNormalizationWithTrueExtraction() throws Exception {
+        Properties props = new Properties();
+        props.setProperty( PipelineConfigurator.USE_POS.key, Configurator.TRUE );
```

Comment:
when you update the cogcomp-nlp dependency version numbers, check this: I believe it is now fixed so that use sentence pipeline can be set to "TRUE" if you want to use it. Remove the TODO if this is indeed the case.